docker/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+
+services:
+  server:
+    image: nginx:1.21.6
+    restart: unless-stopped
+    volumes:
+      - ./data/dist:/usr/share/nginx/html:ro
```

Comment:
Why not just a docker volume instead of dealing with absolute paths?

Reply:
Good point, I just normally do it that way because it makes backups more convenient.

Reply:
True. Whoever i don't think this should count as persistent data